torchaudio/backend/common.py

```diff
@@ -72,6 +89,9 @@ def __init__(self,
 
 _LOAD_DOCSTRING = r"""Loads an audio file from disk into a tensor
 
+Warnings:
+    This function is deprecated. Please migrate to the new backend.
```

Comment:
"... is deprecated and will be removed in a future release."

"new backend" should be replaced with a more specific recommendation.